types/webpack-dev-server/webpack-dev-server-tests.ts

```diff
@@ -273,6 +273,39 @@ server = new WebpackDevServer(
             publicPath: '/assets/',
         },
         https: true,
+        server: {
+            type: 'https',
+            options: {
+                passphrase: '----',
+            },
+        },
+    },
+    compiler,
+);
+
+server = new WebpackDevServer(
+    {
+        server: 'spdy',
```

Comment:
Please put the new options in the existing webpack dev server configs above (`c2`--`c6`).